pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.1.13"
+version = "0.2.0"
```

Comment:
did we ever agree on a versioining scheme? I'm not opposed to this bump, I'm just curious

Reply:
I would say this scheme generally supports the semantic versioning definition of a minor rev.